### ISSUE TITLE
Annoying text display bug

### ISSUE THREAD

**Issue by m-ibm** [alexandersimoes/d3plus#357]
The screen shots below show the top-left rectangle of a treemap and a bit of its surroundings.

When the rectangle is a certain size, this happens:
![display_bug](https://cloud.githubusercontent.com/assets/13468737/8852424/c24f028a-315d-11e5-98c5-2bdb6bbafd70.png)
The text displayed in the rectangle is wrong! ... see the tool tip for the correct text.

Below is what should **always** be displayed:
![display_bug 2](https://cloud.githubusercontent.com/assets/13468737/8852427/c6a77bf0-315d-11e5-9362-1a23ba26f14c.png)

**Comment by davelandry** [comment]
I'm having trouble recreated this. Can you confirm that you're using the latest release, version 1.7.4?

You can check by typing this into the browser console:
```js
d3plus.version
```

**Comment by m-ibm** [comment]
The version I am currently using is 1.6.9.
I prefer (most) of its default color scheme and the way it displays initialization messages.

I did try the latest (1.7.4) and I still get the issue.

I don't know if it helps, but I see this when moving from a 9x16 screen to a 16x9 screen (my secondary display is upside-down).

**Comment by davelandry** [comment]
The latest version is 1.8.0, please try that version and see if the issue is resolved.

If not, can you post a jsfiddle of your code displaying the error? I need the specific visualization's data and width/height to recreate it.

**Comment by davelandry** [comment]
@m-ibm have you been seeing this issue with the latest release? If not, I'm going to close this issue as resolved.

**Comment by m-ibm** [comment]
No, I haven't seen it with the latest release (1.9.0), albeit, I cannot seem to locate the data I used when creating the post - so it doesn't mean the bug isn't there, I just cannot recreate it at the moment.

I do see quite a few usability issues. I will open a separate issue with those.
